### PR TITLE
context in intercept questions, default port 80

### PR DESCRIPTION
**Pull request by njayp**
Signed-off-by: njayp <nickjaypowell@gmail.com>

## Description

The questions asked for a telepresence preview url were made less technical. The default port was changed from undefined to 80

## Checklist

 - [x] I made sure to update `./CHANGELOG.md`.
 - [ ] I made sure to either submit a docs PR, or tell Matt about the necessary documentation changes.
 - [ ] My change is adequately tested.
 - [ ] I updated `DEVELOPING.md` with any any special dev tricks I had to use to work on this code efficiently.

### CODE REVIEW

**Review by thallgren — APPROVED** [review]
LGTM. Would it make sense to also give a \<service name>.\<service namespace> as the default for the ingress IP address?

**Comment by thallgren** [comment]
This PR also solves the "less technical ingress" issue. It should be noted in the description (and included in the changelog).

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.4.4 (TBD)
+
+- Feature: When creating a preview url, the default port is now 80 rather than undefined
+
 ### 2.4.3 (September 15, 2021)
```
Comment (thallgren) [inline CHANGELOG.md]: Please also add an entry about the "less technical ingress".

**Review by thallgren — CHANGES_REQUESTED** [review]
Please also add a description to the PR. It's not uncommon to search for PRs using keywords.

**Review by josecv — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +32,8 @@
 
 - Bugfix: Telepresence correctly reports an intercept port conflict instead of panicking with segfault.
 
+- Bugfix: Intercept questions about the cluster ingress were made more intuitive.
```
Comment (josecv) [inline CHANGELOG.md]: should this be here? 2.4.4 is already out

**Review by josecv — APPROVED** [review]
looks good just fix the changelog and it's good to merge

File: CHANGELOG.md (hunk 2 above)
Comment (njayp) [reply CHANGELOG.md]: ha yep deleted

**Pull request merged**